scripts/shared/lib/shflags

```diff
@@ -4,14 +4,16 @@
 
 if [ -f /usr/share/shflags/shflags ]; then
     . /usr/share/shflags/shflags
```

Comment:
I think it'll be much cleaner if you just put exit 0 after this line

Too much indentation makes code harder to read and maintain

Reply:
I don’t think you’ll like it when the sourced `shflags` exits ;-).

Reply:
I’ve moved the logic into a function so it’s clearer, hopefully.

Reply:
Ah right i was thinking of `return` style here but i guess you cant `return` if not in a function..